app/main.py

```diff
@@ -1,3 +1,54 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        print(f"Distance: {self.km} kilometers)")
+
+    def __repr__(self):
+        return f'Distance(km={self.km})'
+
+    def __add__(self, kilometers):
+        if isinstance(kilometers, int) or isinstance(kilometers, float):
```

Comment:
you can shorten this to `isinstance(other, (int, float))`
fix this throughout your code